src/plugins/elasticsearch/index.js

```diff
@@ -59,7 +59,9 @@ module.exports = function (kibana) {
       );
 
       // Set up the health check service and start it.
-      healthCheck(this, server).start();
+      var hc = healthCheck(this, server);
+      server.expose('waitUntilReady', hc.waitUntilReady);
```

Comment:
I feel like we should be using an arrow here, to make sure `hc.waitUntilReady` it doesn't loose it's `this` context